.idea/codeStyles/Project.xml

```diff
@@ -27,6 +27,9 @@
         <ANNO name="org.eclipse.jdt.annotation.Nullable" />
       </REPEAT_ANNOTATIONS>
     </JavaCodeStyleSettings>

```

Comment:
It'd be nice if you could revert this and the other change, I'll be doing it tomorrow either way so don't be worried about it.